openedx/core/djangoapps/courseware_api/views.py

```diff
@@ -338,3 +348,59 @@ def get(self, request, course_key_string, *args, **kwargs):
             pass
 
         return Response(resp)
+
+
+class Celebration(DeveloperErrorViewMixin, APIView):
+    """
+    **Use Cases**
+
+        Marks a particular celebration as complete
+
+    **Example Requests**
+
+        POST /api/courseware/celebration/{course_key}
+
+    **Request Parameters**
+
+        Body consists of the following fields:
+
+            * first_section (bool): whether we should celebrate when a user finishes their first section of a course
+
+    **Returns**
```

Comment:
In your tests, you also specify that a 401 can be returned. We should document that as a potential return code.

Reply:
Fixed.